domains/pallets/domain-registry/src/lib.rs

```diff
@@ -819,7 +819,20 @@ impl<T: Config> Pallet<T> {
 
         let core_block_number = T::BlockNumber::from(*core_block_number);
 
-        if !core_block_number.is_zero() {
+        // Considering this senario, a core domain stalls at block 1 for a long time and then
+        // resumes at block 1000, assuming `MaximumReceiptDrift` is 128, the range of receipts
+        // in the new bundle created at block 1000 would be [1, 1+128], thus the state root
```

Comment:
In code I'd prefer Rust syntax: `1..=1+128` so it is very clear whether right side is inclusive or not.

Reply:
Well, not strictly in the code, but in the comment, `[]` means both inclusive, https://stackoverflow.com/questions/4396290/what-does-this-square-bracket-and-parenthesis-bracket-notation-mean-first1-last